Library/Homebrew/lazy_object.rb

```diff
@@ -1,6 +1,8 @@
 # typed: true
 # frozen_string_literal: true
 
+require "delegate"
```

Comment:
This seems to have previously depended on the `require` within `try` [itself](https://github.com/rails/rails/blob/v6.1.7.6/activesupport/lib/active_support/core_ext/object/try.rb#L3).